drivers/user_permission_report/app/views/user_permission_report/warehouse_reports/users/show.haml

```diff
@@ -0,0 +1,24 @@
+- content_for :modal_title, "Full Permissions for #{@user.name}"
+
+%h1= @user.name
+%h2= @user.email
```

Comment:
email should be moved out to the table, no need to echo it here.

Reply:
✅